crud-cert-auth/src/main/webapp/WEB-INF/web.xml

```diff
@@ -76,5 +71,19 @@ xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://java.sun.com/xml/ns
     <param-name>resteasy.providers</param-name>
     <param-value>com.restcompress.provider.LZFEncodingInterceptor,com.restcompress.provider.LZFDecodingInterceptor</param-value>
   </context-param>
+  
+  <listener>
+    <listener-class>
+      com.redhat.lightblue.rest.crud.cert.auth.health.CrudCertAuthHealthCheckServletContextListener
+    </listener-class>
+  </listener>
+  <servlet>
+    <servlet-name>healthCheck</servlet-name>
+    <servlet-class>com.codahale.metrics.servlets.HealthCheckServlet</servlet-class>
+  </servlet>
+  <servlet-mapping>
+    <servlet-name>healthCheck</servlet-name>
+    <url-pattern>/healthcheck</url-pattern>
```

Comment:
Maybe /v2/health or /health/v2 or /health/complete or something like that? @derek63 what do you think?

Otherwise I feel like "what's the difference between /health and /healthcheck"? This new health endpoint really supersedes the old one.